deploy-odf/deploy.sh

```diff
@@ -84,31 +86,41 @@ if ! ./verify.sh; then
         done
         echo ">>>>>>>>>>>>>>>>>>>>>>>>>>>"
 
-        echo ">>>> Render and apply manifest to deploy ODF StorageCluster"
-        echo ">>>>>>>>>>>>>>>>>>>>>>>>>>>"
-        render_file manifests/04-ODF-StorageCluster.yaml
-        sleep 60
+	if [ "${NUM_M}" -eq "3" ];
+	then
+		echo ">>>> Render and apply manifest to deploy ODF StorageCluster"
+		echo ">>>>>>>>>>>>>>>>>>>>>>>>>>>"
+		render_file manifests/04-ODF-StorageCluster.yaml
+		sleep 60
+
+		echo ">>>> Waiting for: ODF Cluster"
+		echo ">>>>>>>>>>>>>>>>>>>>>>>>>>>>>"
+		timeout=0
+		ready=false
+		while [ "$timeout" -lt "1000" ]; do
+		    if [[ $(oc get --kubeconfig=${EDGE_KUBECONFIG} -n openshift-storage storagecluster -ojsonpath='{.items[*].status.phase}') == "Ready" ]]; then
+			ready=true
+			break
+		    fi
+		    sleep 5
+		    timeout=$((timeout + 1))
+		done
+
+	    sleep 30
+	    oc --kubeconfig=${EDGE_KUBECONFIG} patch storageclass ocs-storagecluster-ceph-rbd -p '{"metadata": {"annotations":{"storageclass.kubernetes.io/is-default-class":"true"}}}'
+	else
+		echo ">>>> Render and apply manifest to deploy ODF StorageSystem"
+		echo ">>>>>>>>>>>>>>>>>>>>>>>>>>>"
+		render_file manifests/04-MCG-StorageCluster.yaml
+		sleep 60
+		ready=true
```

Comment:
Im missing here a check/verification after applying the MCG

Reply:
good catch! Let me add that